bluebrain/repo-bluebrain/packages/py-neurodamus/package.py

```diff
@@ -29,21 +31,25 @@ class PyNeurodamus(PythonPackage):
             description="Add more dependencies to support advanced use cases")
 
     # Note: we depend on Neurodamus but let the user decide which one.
-    # Note: avoid Neuron dependency due to issues with Intel and GCC conflicts.
+    # Note: avoid Neuron/py-mvdtool dependency due to Intel-GCC conflicts.
     depends_on('python@3.4:',      type=('build', 'run'))
     depends_on('py-setuptools',    type=('build', 'run'))
     depends_on('py-h5py',          type='run')
     depends_on('py-numpy',         type='run')
     depends_on('py-docopt',        type='run')
     depends_on('py-libsonata',     type='run', when='@2.5.3:')
     depends_on('py-morphio',       type='run', when='@2.6.0:')
-    depends_on('py-scipy',         type='run', when='+all_deps@2.5.3:')
+    depends_on('py-scipy@1.5.4',   type='run', when='+all_deps@2.5.3:')
```

Comment:
Do we still need that version pin?

Reply:
Nope. Actually this dependency could go away and loaded on demand for LFP and such